tutorials/ios-chatbot/part-2/index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: How to build a conversational app using Cloud Machine Learning APIs (Part 2 of 3)
+description: How to build a conversational app using Cloud Machine Learning APIs (Part 2 of 3).
```

Comment:
Same here. Could use the text you used at the end of Part 1 as the summary description for this Part.